README.md

```diff
@@ -15,13 +15,14 @@ Our _benchit_ CLI allows you to benchmark Pytorch models without changing a sing
 
 <img src="https://github.com/groq/mlagility/raw/main/docs/img/mlagility.gif"  width="800"/>
 
-## Tons of Models
+## 1000+ Models
```

Comment:
Some of these models are duplicates from the other libraries we have. Do we want to remove the duplicates?

Reply:
yeah related to https://github.com/groq/mlagility/pull/217#discussion_r1145373659... wondering if its more than just a confusion and might be a bug